app/gui/src/project-view/composables/navigator.ts

```diff
@@ -149,6 +159,7 @@ export function useNavigator(
 
         if (state.last && longpressTimer) cancelLongpress()
         if (state.last && holdDragStarted) holdDragStarted = false
+        if (state.last && preventContextMenu) setTimeout(() => (preventContextMenu = false), 10)
```

Comment:
What exactly these 10 milliseconds means? It's a time after which second right-click will precent context menu's default?

Let it be a named constant.

Reply:
It's a timeout that's needed due to event ordering. The contextmenu event is immediately raised after the touchend (i.e. handler with `state.last` set). We don't want to have it appear because we just handled a drag action. The timeout causes the menu to be reenabled only after that immediate next handler is rejected, allowing future clicks to open it again.

Reply:
So why 10 miiliseconds, and not 1 or 0?

Reply:
I tried using 0 or other things, but then sometimes (about every other time) the `contextmenu` handler is actually fired after the timer. 10ms works consistently for me and doesn't introduce enough delay to noticeably impact behavior (i.e. block legitimate context menu clicks). I'll add a comment about that.